quickwit/quickwit-indexing/src/source/ingest/mod.rs

```diff
@@ -305,8 +305,8 @@ impl IngestSource {
             Vec<TruncateShardsSubrequest>,
         > = FnvHashMap::default();
 
-        for (shard_id, to_position_exclusive) in truncate_positions {
-            if matches!(to_position_exclusive, Position::Beginning) {
+        for (shard_id, truncate_position) in truncate_positions {
```

Comment:
the to_*_exclusive part was actually helpful.

Reply:
Added doc to `TruncateShardsSubrequest` and renamed to `truncate_up_to_position_inclusive`.